test/integration/bulk-action-integration.ts

```diff
@@ -75,7 +75,7 @@ describe('BulkActions Api', async function () {
     })
 
     test('Publish BulkAction with wrong payload', async () => {
-      const entry = await testEnvironment.getEntry(TestDefaults.entry.testEntryId)
+      const entry = await testEnvironment.getEntry(TestDefaults.entry.testEntryReferenceId)
```

Comment:
This feels like we're overloading the testEntryReferenceId for multiple test suites. Should we instead create a separate testEntry for this suite?

Reply:
good point ^ 👍🏽 -- will create a new one

Reply:
to be clear: in this PR? or in another PR?

Reply:
(did it on this one)